### DISCUSSION TITLE
Cypress docker base image invalid-value

### DISCUSSION THREAD

**Discussion by DawidJanczak** [renovatebot/renovate#18057] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Hello,

Docker images from https://hub.docker.com/r/cypress/browsers don't seem to upgrade correctly, renovate reports "invalid-value" for skip reason. Please find the relevant logs attached.

### Relevant debug logs

<details><summary>Logs</summary>

```
      {
        "packageFile": "cypress/Dockerfile",
        "deps": [
          {
            "depName": "cypress/browsers",
            "currentValue": "node14.17.0-chrome91-ff89",
            "replaceString": "cypress/browsers:node14.17.0-chrome91-ff89",
            "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
            "datasource": "docker",
            "depType": "final",
            "depIndex": 0,
            "updates": [],
            "warnings": [],
            "versioning": "docker",
            "skipReason": "invalid-value"
          }
        ]
      },
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
not a bug. renovate doesn't support that versioning scheme. you probably need to use a custom regex versioning to get it working with renovate

**Reply by DawidJanczak** [reply]
Thank you, I'll try that.

**Comment by BreakBB** [comment]
I am running into the same issue. Can you share a solution for this @DawidJanczak ?

This does not work for me:

```json

        {
            "matchDatasources": [
                "docker"
            ],
            "matchManagers": [
                "dockerfile"
            ],
            "matchPackageNames": [
                "cypress/browsers"
            ],
            "versioning": "regex:^(node(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)(.*))?$"
        }
```

**Reply by DawidJanczak** [reply]
I actually haven't gotten around to it, we occasionally upgrade manually.
